hail/hail-ci-build.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+set -ex
+
+ROOT=$(cd .. && pwd)
+cd hail
```

Comment:
I think `ROOT` is wrong here. we want it to be `/path/to/monorepo/`, right?

Maybe we should maintain the invariant that these files are called from their containing directory? That would move the `cd hail` into the parent's build file. That feels somehow more manageable.

Reply:
Oops, yes, I need to cd.  Good catch.